public/js/reducers/app.js

```diff
@@ -1,9 +1,11 @@
 import * as actionTypes from 'constants/action-types';
+import { getId } from 'utils';
 
 
 export const initialAppState = {
   csrfToken: null,
   error: null,
+  notifications: [],
```

Comment:
It would be nice to comment how this looks will look like `[[key, notice], [key, notice], ...]`